src/main/java/frc/robot/subsystems/trapElvSubsystem/TrapElvSubsystem.java

```diff
@@ -82,16 +82,18 @@ public class TrapElvSubsystem extends SubsystemBase {
   private ElevatorSim m_scoringElevatorSim;
   private SingleJointedArmSim m_wristMotorSim;
 
+  private ShuffleboardTab TrapElvTab = Shuffleboard.getTab(this.getName());
+
   private DebugEntry<Double> currentPositionEntry;
   private DebugEntry<Boolean> isWristRollerRunning;
+  private DebugEntry<Double> FFOutput;
+  private DebugEntry<Double> wristOutput;
+  private DebugEntry<Double> wristGoal;
+  private DebugEntry<String> currentWristStateEntry;
 
-  private ShuffleboardTab TrapElvTab = Shuffleboard.getTab("TrapElvSubsystem");
-  private GenericEntry FFOutput = TrapElvTab.add("FF Output", 0).withPosition(8, 0).getEntry();
-  private GenericEntry wristOutput =
-      TrapElvTab.add("Wrist Motor Output", 0).withPosition(7, 0).getEntry();
-  private GenericEntry wristGoal = TrapElvTab.add("Wrist Goal", 0).withPosition(6, 0).getEntry();
-  private GenericEntry currentWristStateEntry =
-      TrapElvTab.add("Current Wrist State", TrapElvState.STOWED.name()).getEntry();
+  private TunableNumber wristP;
```

Comment:
Should we just lock these numbers in, instead of having tunable numbers?  Thoughts?

Reply:
Could be nice to have is things get weird at comp for some reason